src/Wallabag/CoreBundle/Form/Type/ForgotPasswordType.php

```diff
@@ -20,6 +20,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
             ->add('email', 'email', array(
+                'required' => true,
```

Comment:
We already declare constraints as `new Constraints\NotBlank(),` so this is not required to _extra_ add the `required` attribute.